app/src/openxr/cpp/OpenXRInputMappings.h

```diff
@@ -245,6 +245,20 @@ namespace crow {
                     { kPathHaptic, OpenXRHandFlags::Both },
             },
     };
+
+    // Pico 4 controller: version 5.4.0 of the Pico platform changed the system name
+    const OpenXRInputMapping Pico4v540 = {
+            Pico4.path,
+            "PICO 4",
+            Pico4.systemDoF,
+            Pico4.leftControllerModel,
+            Pico4.rightControllerModel,
+            Pico4.controllerType,
+            Pico4.profiles,
+            Pico4.buttons,
+            Pico4.axes,
+            Pico4.haptics,
+    };
```

Comment:
I am not sure about this. I think it'd be better to adapt the filtering code. You might add some method that usually returns the string that is in the mapping except that for Pico and version <5.4 it'd return "Pico HMD".

Reply:
Done.